File: B78225/orbis/src/main/java/com/orbis/B78225/ventas/domain/Libro.java
package com.orbis.B78225.ventas.domain;

import java.util.List;

public class Libro {

	private int idLibro; 
	private String titulo;
	private int ano;
	private String ISBN;
	private float precio;
	private Editorial editorial;
	private List<Autor> autores;

	public String getISBN() {
		return ISBN;
	}
	public void setISBN(String ISBN) {
		this.ISBN = ISBN;
	}
	public int getIdLibro() {
		return idLibro;
	}
	public void setIdLibro(int idLibro) {
		this.idLibro = idLibro;
	}
	public String getTitulo() {
		return titulo;
	}
	public void setTitulo(String titulo) {
		this.titulo = titulo;
	}
	public int getAno() {
		return ano;
	}
	public void setAno(int ano) {
		this.ano = ano;
	}
	public float getPrecio() {
		return precio;
	}
	public void setPrecio(float precio) {
		this.precio = precio;
	}
	public Editorial getEditorial() {
		return editorial;
	}
	public void setEditorial(Editorial editorial) {
		this.editorial = editorial;
	}
	public List<Autor> getAutores() {
		return autores;
	}
	public void setAutores(List<Autor> autores) {
		this.autores = autores;
	}



}

File: B40894/Laboratorios/Laboratorio4/OrbisB40894/OrbisB40894.Model/Domain/Libro.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace OrbisB40894.Model.Domain
{
    public class Libro
    {
        private int cod_libro;

        public int Cod_libro { get; private set; }
        public string Titulo { get; private set; }
        public int Año_publicacion { get; private set; }
        public string Isbn { get; private set; }
        public double Precio { get; private set; }


        private string titulo;
        private int año_publicacion;
        private string isbn;
        private double precio;

        private List<Autor> autores = new List<Autor>();
        public List<Autor> Autores { get; private set; }

        private Publicador publicador;

        public Publicador Publicador { get; private set; }

        public Libro(int cod_libro, string titulo, double precio)
        {
            
            Titulo = titulo;
            Cod_libro = cod_libro;
            Precio = precio;
        }

    }

}
